mcux/README

```diff
@@ -30,6 +30,7 @@ Status:
    MIMXRT1061	SDK 2.8.2 (2020-08-19)
    MIMXRT1062	SDK 2.8.2 (2020-08-19)
    MIMXRT1064	SDK 2.8.2 (2020-08-19)
+   MIMXRT1170   SDK 2.9.0 (2021-01-29)
```

Comment:
Minot issue, could you please fix the spacing

Reply:
However I don't update device information for other RT soc. I can't update readme for other rt soc because I update the common drivers.